ci-automation/vendor-testing/azure.sh

```diff
@@ -56,6 +56,7 @@ run_kola_tests() {
       --azure-size="${instance_type}" \
       --azure-hyper-v-generation="${hyperv_gen}" \
       ${AZURE_USE_GALLERY} \
+      ${AZURE_KOLA_VNET:+--azure-kola-vnet=${AZURE_KOLA_VNET}} \
```

Comment:
Please also add the (empty) definition of AZURE_KOLA_VNET to `ci-automation/ci-config.env`, so it'll be obvious that it's something to be provided in env.